chokuretsu/guide/play-game.html

```diff
@@ -0,0 +1,222 @@
+    <script src="https://kit.fontawesome.com/aa4c023dda.js" crossorigin="anonymous"></script>
+
+    <!-- Platform detection script -->
+    <script src="guide.js" type="text/javascript"></script>
+</head>
+<body>
+<main id="column-with-sidebar">
+    <meta id="redirect-if-device-not-selected">
+    <div class="column-with-sidebar-body">
+        <div class="one-column-text" id="game-logo-container">
+            <object class="image-graphic drop-shadow" width="100%" data="../assets/game-logo.svg" type="image/svg+xml">
+                <img src="../assets/game-logo.png" alt="The Series of Haruhi Suzumiya logo"/>
+            </object>
+        </div>
+
+        <a href="choose-platform.html">
+            <div class="guide-platform-indicator hidden drop-shadow">
+                <object id="guide-platform-indicator-icon" width="20em" type="image/svg+xml"></object>
+                <b id="guide-platform-indicator-text"></b>
+            </div>
+        </a>
+
+        <div class="one-column-text">
+            <h1>Playing the game</h1>
+            <p>Now that you've got the patched ROM file, it's time to get ready to start playing!</p>
+        </div>
+
+        <!-- Playing on an emulator -->
+        <div class="one-column-text hidden emu-show">
+            <h3>Emulating the game</h3>
+            <p>There are a few options for DS emulators, but the one we recommend is MelonDS for its accuracy,
+                performance and feature set.</p>
+            <p>If you own a Wii U console, Nintendo's official Virtual Console DS emulator provides accurate emulation,
+                complimented nicely by the Wii U Gamepad, which works great with the touchscreen used frequently in
+                this game.</p>
+
+            <h4>MelonDS setup</h4>
+            <b>Requirements:</b>
+            <ul>
+                <li>Windows, macOS or Linux computer</li>
+                <li>Nintendo DS or DS lite console</li>
+                <li>Flash cartridge (e.g. R4 cartridge. Also, an SD card to load stuff onto it, if necessary)</li>
+            </ul>
+            <b>Setup:</b>
+            <ol>
+                <li><a href="https://melonds.kuribo64.net/downloads.php" target="_blank" class="exclude-device-href">Download
+                    MelonDS</a> from the project website. Put this somewhere easily accessible, such as where you put
+                    your applications or documents on your computer.
+                </li>
+                <li>Dump your Nintendo DS firmware, bios7 and bios9 using your flash cartridge. This <a
+                        href="https://www.youtube.com/watch?v=CtxSxmvwaKc" target="_blank"
+                        class="exclude-device-href">video tutorial</a> is pretty excellent at covering how to do this,
+                    otherwise:
+                    <ol>
+                        <li>Download the <a href="https://www.dropbox.com/s/bei4xcrb5gueax0/Firmware%20Dumper.zip?dl=1"
```

Comment:
Perhaps we should host this ourselves rather than relying on a dropbox link? We could create it as a release artifact on this repository?

Reply:
I'm not sure if this is released under a permissive license that would let us do that

Reply:
Perhaps https://archive.org/details/dsbf-dumper would work -- that way we're not hosting it but it's still something less likely to die than a dropbox link, I think.

Reply:
That works - perfect.